server/package.json

```diff
@@ -21,7 +21,8 @@
         "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
         "test:e2e": "jest --config ./test/jest-e2e.json",
         "checks": "npm run test:e2e && npm run test && npm run lint",
-        "prepare": "cd .. && husky install && cd server"
+        "prepare": "cd .. && husky install && cd server",
+        "seed:drop": "ts-node -r tsconfig-paths/register src/database/seeding/seed.ts"
```

Comment:
this command naming to me says we're dropping the db/data
but it's really doing that and re-seeding
maybe `dropAndSeed`?